nyc_taxi/nyc_taxi-nongeo.ipynb

```diff
@@ -192,11 +179,8 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "scatter = hv.Scatter(df, 'passenger_count', 'tip_amount')\n",
-    "ranged = scatter.redim.range(tip_amount=(0,60), passenger_count=(-0.5,6.5))\n",
-    "shaded = hd.spread(hd.datashade(ranged, x_sampling=0.15))\n",
-    "labelled = shaded.redim.label(passenger_count=\"Passengers\", tip_amount=\"Tip, $\")\n",
-    "labelled"
+    "df.hvplot.scatter('passenger_count', 'tip_amount', rasterize=True, cnorm='log',\n",
```

Comment:
Wow it's quite nice to see that this plot actually can be created with hvPlot! I would recommend also setting `y_sampling` (e.g. to `0.5`) so when you hover over the plot, you can see the number of people who tipped between [$x, $x+0.5]. If you make this update, please update the text below to indicate that.

Reply:
I added `y_sampling` but there was no update to the hover info as far as I could tell.

Reply:
Oh I didn't mean this would change the hover info. But by setting `y_sampling` you make the hover info more understandable, because now it's related to a known y-unit.